packages/order/src/services/order-module-service.ts

```diff
@@ -694,20 +694,25 @@ export default class OrderModuleService<
   ): Promise<void>
   async removeLineItems(itemIds: string, sharedContext?: Context): Promise<void>
   async removeLineItems(
-    selector: Partial<OrderTypes.OrderLineItemDTO>,
+    selector: Partial<OrderTypes.FilterableOrderLineItemProps>,
     sharedContext?: Context
   ): Promise<void>
 
   @InjectTransactionManager("baseRepository_")
   async removeLineItems(
```

Comment:
suggestion: should we name it `deleteLineItems` to keep consistency with what is generated under the hood, otherwise both method will be available and sometimes it will be remove or delete

Reply:
I don't know if it is ok to create this breaking change. @olivermrbl ?
Even though Orders v2 will differ a bit from v1 as we incorporate the "Cart" in it.

Reply:
I agree. And maybe we should stick to keep the endpoints compatible to v1 as possible, but not the services/modules.

Reply:
Agreed 👍

Reply:
The `remove[Something]` and `add[Something]` was from before we had the autogenerated methods. I think it makes sense to remove these in favor of delete and create. 

> And maybe we should stick to keep the endpoints compatible to v1 as possible, but not necessarily the services/modules.

Yes, absolutely. The services do not need to be compatible with V1.